### ISSUE TITLE
pydeps 1.12.2 introduces bogus import

### ISSUE THREAD

**Issue by vonschultz** [thebjorn/pydeps#182]
# Setup

```
echo import someimportforillustrationpurposes > a.py
echo import a > b.py
echo import b > c.py
```

# Old behavior / Expected behavior

Using pydeps version 1.11.2, the following works perfectly, reporting that `c` only imports `b`:
```
pydeps --include-missing --show-deps c.py 
```
The graph is `someimportforillustrationpurposes` → `a` → `b` → `c`, with no other dependency arrows.

# New behavior / Unexpected behavior

Using pydeps version 1.12.2, the following works reports that `c.py` imports both `a` and `b`:
```
pydeps --include-missing --show-deps c.py 
```
The graph is `someimportforillustrationpurposes` → `a` → `b` → `c.py` with an extra arrow from `a` to `c.py`.

**Comment by thebjorn** [comment]
@vonschultz thanks for the bug-report. It should be fixed in v1.12.3 available on PyPI.

**Comment by vonschultz** [comment]
Thanks. Looks good.
